utils/db.ts

```diff
@@ -100,6 +100,73 @@ export async function getItemCommentsCount(itemId: string) {
   return count;
 }
 
+interface InitVote {
+  userId: string;
+  itemId: string;
+  voteId?: string;
+}
+
+export interface Vote extends InitVote {
+  id: string;
+  createdAt: Date;
+}
+
+export async function createVote(initVote: InitVote) {
+  let res = { ok: false };
+  while (!res.ok) {
+    const id = crypto.randomUUID();
+    const votesByUserKey = ["vote_by_users", initVote.userId, id];
```

Comment:
```suggestion
    const votesByUserKey = ["votes_by_users", initVote.userId, id];
```
WDYT? Other referencing to this index will need also to be adjusted.

Reply:
LGTM! 

Ya I would replace all `vote_by_users` with `votes_by_users`